ranger/data/scope.sh

```diff
@@ -94,6 +95,28 @@ handle_extension() {
             jq --color-output . "${FILE_PATH}" && exit 5
             python -m json.tool -- "${FILE_PATH}" && exit 5
             ;;
+	## 3D models
+	## OpenSCAD only supports png image output, and ${IMAGE_CACHE_PATH} is
+        ## hardcoded as jpeg. So we make a tempfile.png and just move/rename it 
+	## to jpg This works because image library is smart enough to handle it
```

Comment:
A very minor nitpick: There is a missing period after "jpg" and before "This works".

Reply:
Good spot. Another is that the line above that is indented more.

Reply:
Hmm, looking at it in another diff view it doesn't look indented differently. Probably because of tabs. Make sure all the indentation is spaces.

Reply:
`/%s/\t/        /g` fixed this good, as I was the only one who put dirty tabs into the file.